### ISSUE TITLE
Render address points and addresses

### ISSUE THREAD

**Issue by lxbarth** [openstreetmap/iD#1524]
Address points (screenshots) and addresses aren't rendered currently. This would be great to include to encourage more address mapping.

[![](http://cl.ly/image/060i0v42182U/Screen%20Shot%202013-05-25%20at%2010.25.58%20AM.png)](http://openstreetmap.us/iD/release/#background=Bing&id=w67231791&map=19.10/-77.01914/38.91357)

**Comment by tordans** [comment]
Could the the feature that shows `ref=*` next to a tree(…) node be re-used for this?
So, instead of only showing `ref` next to a node, we also display `addr:housenumber` next to it.

Screenshot: Above a `addr:housenumber-node`, below a `natural=tree + ref=123` node.
<img width="161" alt="Bildschirmfoto 2019-04-13 um 07 37 15" src="https://user-images.githubusercontent.com/111561/56075262-e85c2380-5dbf-11e9-8a2d-3cef6bd82d4a.png">

**Comment by quincylvania** [comment]
I'm okay with just showing the house number and street for features without any other name. I know #3840 had greater ambitions but I think this would be a fine intermediate step.

<img width="807" alt="Screen Shot 2019-04-15 at 1 57 32 PM" src="https://user-images.githubusercontent.com/2046746/56168649-a33b1b80-5f90-11e9-8324-5ba23496a96d.png">

**Comment by Nakaner** [comment]
Rendering the both house numbers and street names adds a lot of labels to the map hiding a lot of other information (background layer). It seems more reasonable to render house numbers only. Otherwise you will have to deal with `addr:place` and other special cases addresses can have.

**Comment by tordans** [comment]
@quincylvania I see a merge commit in your PR. Will it be live at http://preview.ideditor.com/master/#background=Berlin-2018&disable_features=boundaries&map=19.37/52.47279/13.44774 soon to try out (isn't the master synced automatically with "preview"?)

**Comment by quincylvania** [comment]
@tordans Nope, I closed the PR instead of merging. We're not going this direction at this time after all.

**Comment by bagage** [comment]
Thanks @quincylvania for trying a first partial solution! What were the reasons to finally not merge it?

Also #3840 is closed too, so yet there is no plan for solving that issue yet, is that correct?

**Comment by jidanni** [comment]
Odd, are there any other names that carto renders,
https://www.openstreetmap.org/?#map=19/24.23887/120.83488
but iD doesn't?!

**Comment by Filip10** [comment]
Is there some progress on showing house numbers like JOSM or other editors do? A lot of newies are deleting old bulding and replacing them with new one using iDeditor not realizing some of them have address tags. Or is there some replace geometry function like in JOSM to tell it to newies?
